.gitlab-ci.yml

```diff
@@ -400,24 +406,18 @@ build-rustdoc:
   after_script:
     - buildah logout --all
 
-publish-polkadot-image:
+# This image is used in testnets
+# Release image is handled by the Github Action here:
+# .github/workflows/publish-docker-release.yml
+publish-polkadot-debug-image:
   stage:                           build
   <<:                              *build-push-image
+  <<:                              *common-refs
   variables:
     <<:                            *image-variables
     # scripts/dockerfiles/polkadot_injected_debug.Dockerfile
     DOCKERFILE:                    dockerfiles/polkadot_injected_debug.Dockerfile
     IMAGE_NAME:                    docker.io/paritypr/synth-wave
-  rules:
-    # Don't run on releases - this is handled by the Github Action here:
-    # .github/workflows/publish-docker-release.yml
-    - if: $CI_PIPELINE_SOURCE == "pipeline"
-      when: never
```

Comment:
this was to avoid running the job i.e. when `paritytech/scrips` builds a CI image and triggers this CI.